inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -152,6 +150,15 @@ add_library(${TARGET_NAME} SHARED
             ${vs_version_file}
             $<TARGET_OBJECTS:${TARGET_NAME}_obj>)
 
+foreach(src IN LISTS IE_STATIC_DEPENDENT_FILES LIBRARY_SRC LIBRARY_HEADERS PUBLIC_HEADERS)
+    if(IS_ABSOLUTE "${src}")
+        list(APPEND full_src_names ${src})
+    else()
+        list(APPEND full_src_names "${CMAKE_CURRENT_SOURCE_DIR}/${src}")
```

Comment:
Can we use full path from the beginning?

Reply:
Fixed